Library/Formula/ghc.rb

```diff
@@ -1,8 +1,8 @@
 class Ghc < Formula
   desc "Glorious Glasgow Haskell Compilation System"
   homepage "https://haskell.org/ghc/"
-  url "https://downloads.haskell.org/~ghc/7.10.3/ghc-7.10.3a-src.tar.bz2"
-  sha256 "877899988b64d13a86148f4db29b26d5adddef9207f718b726dc5c202d8efd8e"
+  url "https://downloads.haskell.org/~ghc/7.10.3/ghc-7.10.3b-src.tar.bz2"
```

Comment:
Is this stable? Can't find any reference to it on the upstream website.

Note the `a` release here doesn't signify alpha; the first 7.10.3 release they cut lacked some documentation so they tagged a new baby release. Alphabet releases aren't the norm.